packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2727,6 +2774,7 @@ abstract class RenderObject extends AbstractNode implements HitTestTarget {
       description.add('layer: $_layer');
     if (_semantics != null)
       description.add('semantics: $_semantics');
+    description.add('isSemanticBoundary: $isSemanticBoundary');
```

Comment:
Rather than always including this (which will make the output verbose and repetitive), I would have:

```dart
if (isBlockingSemanticsOfPreviouslyPaintedNodes)
  description.add('blocks semantics of earlier render objects below the common boundary');
if (isSemanticBoundary)
  description.add('semantic boundary');
```